modules/tinymce/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Dialogs will not exceed the window height on smaller screens #TINY-8146
 - Text alignment could not be applied to `pre` elements #TINY-7715
+- delete selection on `InsertParagraph` and `mceInsertNewLine` execution #TINY-8606
```

Comment:
```suggestion
- The selection was not deleted when using the `InsertParagraph` or `mceInsertNewLine` commands #TINY-8606
```
We need to say what the bug was that was fixed more so than what was changed here.

Reply:
should we also add a reason? something like:
```
The selection was not deleted when using the `InsertParagraph` or `mceInsertNewLine` commands in order to make the behavior like when `insertParagraph` is called via browser native API (`document.execCommand()`)
```

Reply:
We could but that's far too long as the entries also need to be reasonably short. In this case I don't think it's needed though. If we did still want to, then maybe do something like this?
```suggestion
- The `InsertParagraph` or `mceInsertNewLine` commands did not delete the current selection like the native command used to #TINY-8606
```

Reply:
ok, done, thanks